src/formatting_stack/git_status_formatter.clj

```diff
@@ -0,0 +1,135 @@
+(ns formatting-stack.git-status-formatter
+  "A set of defaults apt for frequent usage XXX.
+
+  See also: `formatting-stack.branch-formatter`, `formatting-stack.project-formatter`"
+  (:require
+   [formatting-stack.core]
+   [formatting-stack.formatters.clean-ns :as formatters.clean-ns]
+   [formatting-stack.formatters.cljfmt :as formatters.cljfmt]
+   [formatting-stack.formatters.how-to-ns :as formatters.how-to-ns]
+   [formatting-stack.formatters.newlines :as formatters.newlines]
+   [formatting-stack.formatters.no-extra-blank-lines :as formatters.no-extra-blank-lines]
+   [formatting-stack.formatters.trivial-ns-duplicates :as formatters.trivial-ns-duplicates]
+   [formatting-stack.linters.eastwood :as linters.eastwood]
+   [formatting-stack.linters.kondo :as linters.kondo]
+   [formatting-stack.linters.line-length :as linters.line-length]
+   [formatting-stack.linters.loc-per-ns :as linters.loc-per-ns]
+   [formatting-stack.linters.ns-aliases :as linters.ns-aliases]
+   [formatting-stack.linters.one-resource-per-ns :as linters.one-resource-per-ns]
+   [formatting-stack.processors.cider :as processors.cider]
+   [formatting-stack.protocols.formatter :as protocols.formatter]
+   [formatting-stack.protocols.linter :as protocols.linter]
+   [formatting-stack.protocols.processor :as protocols.processor]
+   [formatting-stack.reporters.pretty-printer :as pretty-printer]
+   [formatting-stack.strategies :as strategies]
+   [medley.core :refer [deep-merge]]
+   [nedap.speced.def :as speced]
+   [nedap.utils.modular.api :refer [implement]]))
+
+(speced/defn default-formatters [_
+                                 ^vector? formatters-strategies
+                                 ^map? third-party-indent-specs]
```

Comment:
Let's add a default for `third-party-indent-specs` in `cljfmt/new and `cider/new`. Then we can remove the references to `third-party-indent-specs`.

Consumers should overwrite the entire member if they want to change the specs. (which is how other members should work anyway). wdyt?